app/views/profile/profile.html.erb

```diff
@@ -66,7 +66,7 @@
         title: t('titles.click_to_change_profile_pic', "Click to change profile pic"),
         sr_content: t('Click to change profile picture for %{display_name}', :display_name => @user.short_name)) %>
   <% else %>
-    <%= avatar(@user, url: nil, edit: false) %>
+    <%= avatar(@user, url: nil, edit: false, class: "piofile_pic") %>
```

Comment:
Oops, it looks like we have a typo here. We want this to be

`class: "profile_pic"`

, correct?

Reply:
Yes, you're right. It's my mistake.